espressif-devcon/2023/2023.md

```diff
@@ -17,15 +17,180 @@ headingDivider: 1
 
 Last year's talk: [EDC22 Day 1 Talk 7: Rust on Espressif chips](https://www.youtube.com/watch?v=qeEmJ_-6fPg).
 
-`async` in Rust adds two new keywords to the language, `async` & `await`, where 
+`async` in Rust adds two new keywords to the language, `async` & `await`, where
 
  - `async` defines a block or function to be asynchronous
  - `await` defines yield points _within_ an `async` block or function.
 
+## async/.await
 
-## Why use async?
+- building blocks for async blocks, closures, and functions that can be paused and yield control back to the caller
+- `async` defines blocks, closures, and functions to be asynchronous
+- `await` defines a yield point within `async` code
+- `async` code returns a value that implements the `Future` trait
```

Comment:
I'm a bit confused about this line. Is this trying to say that an async function its transpiled to a future by the compiler? Or is it trying to say something else?